mapie/regression/time_series_regression.py

```diff
@@ -299,17 +299,20 @@ def adapt_conformal_inference(
         X, y = cast(NDArray, X), cast(NDArray, y)
 
         self._get_alpha()
-        alpha = cast(Optional[NDArray], check_alpha(alpha))
```

Comment:
Shouldn't we use `check_alpha` on `confidence_level`?
Not so much for the transformation, but at least for the checks.

Reply:
Also, maybe we should use confidence_level_np instead of alpha_np, because we end up using confidence level with .predict